frontend/encore/postcss.rst

```diff
@@ -24,6 +24,21 @@ Next, create a ``postcss.config.js`` file at the root of your project:
         }
     }
 
+Alternatively, you can add the following lines to your ``package.json`` file to avoid creating a ``postcss.config.js`` file:
```

Comment:
Can we please double check if `package.json` is correct and you don't want to refer to `webpack.config.js` instead? Thanks.